File: apphost.cs
#:package CommunityToolkit.Aspire.Hosting.Bun@13.1.2-beta.516
#:sdk Aspire.AppHost.Sdk@13.3.0-preview.1.26126.7

var builder = DistributedApplication.CreateBuilder(args);

var web = builder.AddBunApp(
    name: "psdb-site",
    workingDirectory: "./src/philasdb.site",
    entryPoint: "dev")
    .WithHttpEndpoint(targetPort: 4321);

builder.Build().Run();

File: apphost.ts
import { createBuilder } from "./.modules/aspire.js";

const builder = await createBuilder();

const web = await builder
  .addExecutable("psdb-site", "bun", "./src/philasdb.site", ["run", "dev"])
  .withHttpEndpoint({ targetPort: 4321 });

await builder.build().run();
